src/bridge/ICommon.sol

```diff
@@ -0,0 +1,21 @@
+// Copyright 2021-2022, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro-contracts/blob/main/LICENSE
+// SPDX-License-Identifier: BUSL-1.1
+
+// solhint-disable-next-line compiler-version
+pragma solidity >=0.6.9 <0.9.0;
+
+interface ICommon {
```

Comment:
the name is a bit confusing, why don't we just have these in IBridge?

Reply:
removed and put in IBridge